tls/s2n_record_write.c

```diff
@@ -63,6 +63,11 @@ static uint16_t overhead(struct s2n_connection *conn)
         extra += active->cipher_suite->record_alg->cipher->io.comp.record_iv_size;
     }
 
+    /* TLS 1.3 protected record occupies one extra byte for content type */
```

Comment:
while you are in here, care to rename the function to something more descriptive, for example:

`s2n_calculate_write_overhead`

?

Reply:
renamed to s2n_tls_record_overhead() :)